packages/react/src/views/Message/MessageToolbox.js

```diff
@@ -120,7 +120,7 @@ export const MessageToolbox = ({
         id: 'edit',
         onClick: () => handleEditMessage(message),
         iconName: 'edit',
-        visible: message.u._id === authenticatedUserId,
+        visible: (message.u._id === authenticatedUserId) && (message.files?.[0].type !== 'audio/mpeg') && (message.files?.[0].type !== 'video/mp4'),
```

Comment:
But not all audio and video are in MPEG/MP4 format, right?

We might need to create a mapping for suitable formats and then try

Reply:
Hello Zishan, Currently all our audios are of type: 'audio/mpeg' & videos are of type: 'video/mp4'

These values are hardcoded in the 'AudioMessageRecorder.js' and 'VideoMessageRecoder.js' files.

![image](https://github.com/user-attachments/assets/e8d892e8-43cb-4c58-bb41-138fedcddbf5)

![image](https://github.com/user-attachments/assets/ad985b53-8d20-4daf-8ec5-5e670add7788)


So, I think the above condition will work for all audio and video files since we currently have only one format. 

If we later introduce new formats for audio and video, we will update this logic accordingly.

Reply:
Okay, this is only for the audio and video message recorded directly ? 

And not for the file sent ?

Reply:
Yes, Correct.